database/index.js

```diff
@@ -0,0 +1,58 @@
+const user = '<postgres user>'
+const host = 'localhost'
+const database = '<postgres db name>'
+const password = '<postgres password>'
+const port = '<postgres port>'
+
+import { Sequelize, Model, DataTypes } from 'sequelize'
+// require(sequelize)
+
+const db = new Sequelize({
+    database: 'flick_db',
+    username: 'samuelbelete',
```

Comment:
let's save username and stuff as `.env` variables

Reply:
With Ben on this.